src/components/skeletons/ProductCardSkeleton/product-card-skeleton.scss

```
@@ -0,0 +1,72 @@
+@import "../../../styles/vendors/include-media.scss";
```

```suggestion
@import "../../../styles/scaffold";
```
so that we can use the utilities as well